### PR TITLE
Setup eslint (#91)

### LINKED ISSUE #91: Add Linter to the project
Pull request #93

### PR DESCRIPTION
**Pull request by feekosta**
Close #91

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,5 +33,22 @@
...
+    "eslint-config-airbnb": "^18.1.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-jest": "^23.13.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^3.1.3",
+    "eslint-plugin-react": "^7.20.0",
+    "eslint-plugin-react-hooks": "^2.5.0",
+    "prettier": "^2.0.5",
+    "typescript": "^3.9.3"
```
Comment (miltonbsn) [inline package.json]: Is it necessary the typescript dependency here?

**Comment by miltonbsn** [comment]
As we will work simultaneously on packge.json, wouldn't be a good idea to put the package-lock.json on .gitignore until things be stable to avoid this conflict?

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+	"extends": ["airbnb", "plugin:prettier/recommended", "plugin:jest/recommended"],
+	"rules": {
+		"react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+		"class-methods-use-this": 0,
+		"react/prefer-stateless-function": 0,
+		"react/no-did-update-set-state": 0,
+		"import/no-unresolved": 0,
```
Comment (miltonbsn) [inline .eslintrc]: Are these rules necessary since we don't have react around here?

Comment (feekosta) [reply .eslintrc]: I changed the airbnb dependency to airbnb-base and removed all dependencies related to react

File: package.json (hunk 1 above)
Comment (feekosta) [reply package.json]: it was necessary for an airbnb dependency, but I changed it to airbnb-base, and it is no longer needed

**Review by miltonbsn — APPROVED** [review]

**Comment by idmarjr** [comment]
@miltonbsn , @feekosta 

I took the freedom to merge master branch into this branch and fix conflicts in package.json so we can move forward merging it into master.

Can some of you please review and merge if everything is working fine?

**Comment by feekosta** [comment]
> @miltonbsn , @feekosta
> 
> I took the freedom to merge master branch into this branch and fix conflicts in package.json so we can move forward merging it into master.
> 
> Can some of you please review and merge if everything is working fine?

@idmarjr any other rules of linter that you would indicate? I usually use the rules of airbnb and prettier

**Comment by idmarjr** [comment]
> @idmarjr any other rules of linter that you would indicate? I usually use the rules of airbnb and prettier

Nope. Usually I use a custom version of AirBnB as well.
We can tweak linter rules on the flight if needed. :)

Just make sure to add any setup instructions (If needed) to the project Readme.md

**Pull request merged**